sys-libs/minizip-ng/minizip_ng-4.0.7.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="Fork of the popular zip manipulation library found in the zlib distribution"
+DESCRIPTION="minizip-ng is a zip manipulation library written in C that is supported on Windows, \
+macOS, and Linux."
+HOMEPAGE="https://github.com/zlib-ng/minizip-ng"
+COPYRIGHT="1995-2013 Jean-loup Gailly and Mark Adler"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/zlib-ng/minizip-ng/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="a87f1f734f97095fe1ef0018217c149d53d0f78438bcb77af38adc21dff2dfbc"
+SOURCE_FILENAME="minizip-ng-$portVersion.tar.gz"
+SOURCE_DIR="minizip-ng-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	minizip_ng$secondaryArchSuffix = $portVersion
+	lib:libminizip_ng$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libzstd$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	minizip_ng${secondaryArchSuffix}_devel = $portVersion
+	devel:libminizip_ng$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	minizip_ng$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libzstd$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ninja
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage minizip_ng$secondaryArchSuffix \
+	$libDir/libminizip-ng.so.$libVersion
+
+BUILD()
+{
+	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release \
+		$cmakeDirArgs -GNinja \
+		-DMZ_LIB_SUFFIX="-ng" \
+		-DBUILD_SHARED_LIBS=ON
+	ninja -Cbuild $jobArgs
+}
+
+INSTALL()
+{
+	ninja -Cbuild install
+
+	prepareInstalledDevelLib \
+		libminizip-ng
+	fixPkgconfig
+
+	packageEntries devel \
+		$developDir
+}
+
+TEST()
+{
+	make check
```

Comment:
This is probably wrong when using CMake with Ninja to build it. I would expect something like `ninja -Cbuild test`.

Reply:
No test done in there, this was fixed in: https://github.com/haikuports/haikuports/pull/10613 :)